plugins/plugin-site/gatsby-config.mjs

```diff
@@ -61,7 +61,8 @@ config.plugins = [
                 '@import \'./styles/base.css\';',
                 '@import \'./styles/font-icons.css\';',
                 '@import \'github-syntax-light/lib/github-light.css\';',
-                '@import \'github-syntax-dark/lib/github-auto.css\';'
+                '@import \'github-syntax-dark/lib/github-auto.css\';',
+                '@import \'https://www.jenkins.io/stylesheets/styles.css\';'
```

Comment:
This stylesheet should be loaded by default, doesn't have to be part of extra CSS

Reply:
Ah fantastic - I hadn't realised that.